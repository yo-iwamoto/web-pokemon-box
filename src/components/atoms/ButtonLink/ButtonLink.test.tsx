import { ButtonLink } from '.';
import { render } from '@testing-library/react';

describe('ButtonLink', () => {
  it('正しく表示されること', () => {
    const { getByRole, asFragment } = render(<ButtonLink href='/'>to Top</ButtonLink>);

    expect(getByRole('link', { name: 'to Top' })).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
