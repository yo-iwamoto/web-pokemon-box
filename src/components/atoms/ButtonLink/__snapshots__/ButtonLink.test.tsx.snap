// Vitest Snapshot v1

exports[`ButtonLink > 正しく表示されること 1`] = `
<DocumentFragment>
  <a
    class="rounded bg-blue-500 py-2 px-4 font-bold text-white transition-colors hover:bg-blue-400"
    href="/"
  >
    to Top
  </a>
</DocumentFragment>
`;
